docs/dg/dev/guidelines/project-development-guidelines.md

```diff
@@ -81,7 +83,44 @@ The following tools can help you make your code maintainable:
 - [Code Style Sniffer](/docs/dg/dev/sdks/sdk/development-tools/code-sniffer.html)
 - [PHP Mess detector](https://github.com/spryker/architecture-sniffer)
 
+## Don't be slowed down by Spryker's core rules
+Spryker core development has a lot of rules that are not always applicable to the project level. If you find that a rule is slowing you down, you are not forced to use it. But pay attention that some of the rules are not just recommendations, but conventions and must be followed in order to get a working functionality.
```

Comment:
Do we have a section explaining which rules are fine to be ignored?

Reply:
In Arch Conventions page (that is attached in Apply coding guidelines section) mention rules that are core specific only as I know